src/sections/gitops/PerformanceManagementPage.js

```diff
@@ -87,7 +87,7 @@ const PerformanceManagementPage = () => {
       <ContentContainerWrapper>
         <ContentRow>
           <TextColumn>
-            <Heading>See this sample GitHub workflow (action.yml) for more configuration details.</Heading>
+            <Heading>See this sample GitHub workflow in the 'action.yml' for configuration details.</Heading>
```

Comment:
Thanks, but where is the action.yml link?

Reply:
If not a link then, a local example.

If neither of these, then a different message in this section is needed.

Reply:
I was confused about which action.yaml file you were referring to, although I found this [action.yaml](https://github.com/layer5labs/meshmap-snapshot/blob/master/action.yml) file, please confirm if it is the one you are referring to.